datahub-web-react/src/utils/customGraphQL/useGetEntitySearchResults.ts

```diff
@@ -0,0 +1,30 @@
+import { useMemo } from 'react';
+import { EntityType, SearchInput } from '../../types.generated';
+import { useGetSearchResultsQuery } from '../../graphql/search.generated';
+
+type AllEntityInput<T, K> = Pick<T, Exclude<keyof T, keyof K>> & K;
+
+export function useGetEntitySearchResults(
+    input: AllEntityInput<SearchInput, { type?: EntityType }>,
+    searchTypes: Array<EntityType>,
+) {
+    const result: any = {};
```

Comment:
does this type need to be `any`? Using `any` is going to put callers at risk of throwing javascript errors

Reply:
agree, i will fix that

Reply:
Done